inc/az_ulib_descriptor_api.h

```diff
@@ -5,6 +5,7 @@
 #ifndef AZ_ULIB_DESCRIPTOR_H
 #define AZ_ULIB_DESCRIPTOR_H
 
+#include "az_core.h"
```

Comment:
these are usually included with the directory prefixes:
https://github.com/Azure/azure-sdk-for-c/blob/7ca1b27d127262b501f517ed1fca995371f6ca34/sdk/inc/azure/iot/az_iot_hub_client.h#L20-L22

in this case you could maybe make it
```suggestion
#include "azure/az_core.h"
```